### PR TITLE
11th Small Improvements

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Gave claims on Gnomeregan to Mechagon king.
- Added chance of spawning cannibal characters for some cultures like Ogre, Centaur, Magnataur, Troll (not Zandalari), Qiraji and many others. Character with such "cannibal" cultures are more likely to get "Cannibal" trait via "Carousing" focus and Seclusion events.
- More factors matter when it comes to the class random level up. Ambitious trait increases the chance. Diseases, Fat, Content, Malnourished traits decreases it.
- Nerfed Tribal Monarchy modifiers a bit to make it more like a intermediate state between Monarchy and Tribe.
- Added decision to form the Kor'kron.
- Now you can become Vampire or Death Knight by using Ask for Dark Gift, ability of the Cult of the Damned.
- Stormwind no longer can create the Alliance at the start of the first bookmark.
- Tribal Monarchy barons now become Monarchy when their liege is reformed.
- Swapped current and changed Fel and Void corruption in event tooltips.
- Gave titular duchy to Medivh to make him less likely to be vassalized by kings.
- Purged Entrance to Azjol-Nerub province modifiers.
- Immortal characters are less likely to get Rabies.
- Changed color of Kingdom of Booty Bay to more visible color on the map.
- Vassals of the Horde and the Alliance can create new kingdoms via Found a New Kingdom decision.

Secret changelog:
- Purged vanilla titles in CBs.
- Changed a few human, satyr and quilboar names.
- WoL.7060 can fire for Holy Light, Naaru, Titanic and other religions that have heresies.
- Tidemother and Light group religions can get "the Apostle", "the Confessor", "the Blessed" nicknames via WoL.7070
- In WoL.7091, such option as "I must curb my lust" is disabled if your religion prefers lustful characters (Shath'yar, Legion and Death God religions do so).
- In WoL.7091, such option as "My loins are on fire! Praise [Root.Religion.GetRandomGodName]!" is disabled if your religion prefers chaste characters (Holy Light, Kaldorei, Scarlet Light religions do so).
- Purged a veeery long tooltip after the Reawakening of C'thun if you play the Qiraji.
- Changed adjectives of kingdoms and empires (used in army and fleet names): Gilneasian to Gilnean, Stromgardean to Stromic, Kul Tirasian to Kul Tiran.

### CODE REVIEW

File: localisation/00_zuwc_united_localisation.csv [hunk 1]
```diff
@@ -2227,7 +2227,7 @@ has_unpleasant_trait_trigger_tooltip;Has unpleasant trait (deceitful, shy, paran
 is_my_educator_tooltip;Is my educator\n;;;;;;;;;;;;;x
 has_not_3_students_tooltip;Has not 3 students\n;;;;;;;;;;;;;x
 war_fires_by_event_tooltip;Is war target of event\n;;;;;;;;;;;;;x
-centaur_group_opinion;Centaur Rivalry Opinion;;;;;;;;;;;;;x
+centaur_group_opinion;Centaur Rivalry Opinion ;;;;;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_zuwc_united_localisation.csv]: Is there any reason why you added spaces to a lot of localization lines?

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: No, they appeared after the merge and I don't really know why. Better check this merge again.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/665/commits/b931ab83721c7e5547626b19c8484d4299768af6

Comment (zumbak04) [reply localisation/00_zuwc_united_localisation.csv]: Fixed!

**Comment by MemeWizard** [comment]
>Added decision to form the Kor'kron.

How about use `show_scope_change = no` and don't double vassals names in the tooltip?

<details>

image

</details>

Kor'kron don't become the Horde vassal
<details>

image

</details>

**Comment by MemeWizard** [comment]
>Purged a veeery long tooltip after the Reawakening of C'thun if you play the Qiraji.

Well, if you think it's ok, then it's ok.
<details>

image

</details>

**Comment by zumbak04** [comment]
> > Added decision to form the Kor'kron.
> 
> How about use `show_scope_change = no` and don't double vassals names in the tooltip?

Done!

> Kor'kron don't become the Horde vassal

Fixed!

> > Purged a veeery long tooltip after the Reawakening of C'thun if you play the Qiraji.
> 
> Well, if you think it's ok, then it's ok.

Shortened more!

**Comment by geffe1** [comment]
>Added chance of spawning cannibal characters for some cultures like Ogre, Centaur, Magnataur, Troll (not Zandalari), Qiraji and many others. Character with such "cannibal" cultures are more likely to get "Cannibal" trait via "Carousing" focus and Seclusion events.

Tried it on both male and female characters being summoned, they can get the trait- however I noticed that it is way more likely for a male to get it. At least that was my experience from it. Either way, it works.

**Comment by geffe1** [comment]
ck2_332
Everything works fine, however- would it be a possibility to nerf it down from 35 to perhaps 25 provinces? I feel as if 25 would be a small kingdom.

**Comment by MemeWizard** [comment]
>Everything works fine, however- would it be a possibility to nerf it down from 35 to perhaps 25 provinces? I feel as if 25 would be a small kingdom.

It's realm size, not provinces.
35 holdings ( Realm Size ) ≈ 11 provinces ≈ 3 Duchies ≈ Kingdom

**Pull request merged**